copy-as-format.el

```diff
@@ -180,10 +193,10 @@ work with your processor add the appropriate mapping here.")
 
 (defun copy-as-format--jira (text multiline)
   (if multiline
-      (let ((lang (car (assoc (copy-as-format--language)
-			      copy-as-format--jira-supported-languages))))
+      (let ((lang (car (cdr (assoc (copy-as-format--language)
```

Comment:
Why is this `cdr` necessary?

Reply:
If I understand the purpose of `copy-as-format--jira-supported-languages` correctly, it is to map file extensions on the left hand side to Jira language tags on the right hand side.  Without this `cdr`, I found that the function mapped the left hand side back to the left hand side.

Reply:
Got it. So my original code has a bug :). 

But seems that JIRA accepts `js` and `javascript` for example. Maybe alist is not necessary but that's a problem for another day.

Reply:
> But seems that JIRA accepts js and javascript for example...

Ah yes, you point this out above. But we can leave it for now.